packages/web/src/pages/artist-dashboard-page/store/selectors.ts

```diff
@@ -9,15 +9,13 @@ export const getDashboardTracksStatus = (state: AppState) =>
   state.dashboard.tracksStatus
 export const getDashboardStatus = (state: AppState) => state.dashboard.status
 export const getDashboardTracks = (state: AppState) => state.dashboard.tracks
-export const getUnlistedDashboardTracks = (state: AppState) =>
-  state.dashboard.unlistedTracks
 export const getDashboardListenData = (state: AppState) =>
   state.dashboard.listenData
 
 export const makeGetDashboard = () => {
   return createSelector(
-    [getAccountUser, getDashboardTracks, getUnlistedDashboardTracks],
-    (account, tracks, unlistedTracks) => {
+    [getAccountUser, getDashboardTracks],
```

Comment:
yeah we had all the data in two parts of the reducer before but figured since we need to know a lot about which pills to show in the component, it made sense to move over there